spec/element_spec.rb

```diff
@@ -97,6 +105,14 @@ class PageWithElement < SitePrism::Page
     expect(page).to respond_to :wait_for_some_slow_element
   end
 
+  it 'should be able to wait for an element defined with xpath selector to not exist' do
+    class PageWithElement < SitePrism::Page
+      element :slow_element, :xpath, '//a[@class="slow"]'
+    end
+    page = PageWithElement.new
+    expect(page).to respond_to :wait_for_no_slow_element
```

Comment:
Can we wrap `respond_to` in parentheses, as we're trying to standardise this across the suite. Ideally I'd like only space to be used for `be`